addons/account/i18n/vi.po

```diff
@@ -11833,7 +11833,7 @@ msgstr "Lựa chọn"
 #: model:ir.model.fields.selection,name:account.selection__account_payment__payment_type__outbound
 #: model_terms:ir.ui.view,arch_db:account.account_invoice_send_wizard_form
 msgid "Send"
-msgstr "Gửi"
+msgstr "Chi tiền"
```

Comment:
- Ở đây phần dịch sẽ bị ảnh hưởng đến 2 chỗ: giá trị của trường `payment_type` và string của nút `Send and Print` của wizard account_invoice_send_wizard_form.
- Vì vậy mà em đã làm 1 PR để ghi đè string của nút `Send and Print` trong trường hợp string của nó là 'Send' để dịch lại cho đúng nghĩa: https://github.com/Viindoo/tvtmaaddons/pull/6926

Reply:
@tuoi1501 tách ra làm 2 dòng như này giúp anh nhé
```
#. module: account
#: model:ir.model.fields.selection,name:account.selection__account_payment__payment_type__outbound
msgid "Send"
msgstr "Chi tiền"

#. module: account
#: model_terms:ir.ui.view,arch_db:account.account_invoice_send_wizard_form
msgid "Send"
msgstr "Gửi"
```

Reply:
@royleviindoo cái này em đã thử tách rồi ạ, nhưng khi chạy trên local thì báo lỗi ạ

Reply:
@tuoi1501 lỗi gì em?

Reply:
> @tuoi1501 lỗi gì em?

@royleviindoo Khi mà em cập nhật lại bản dịch thì hệ thống báo lỗi này ạ
![image](https://user-images.githubusercontent.com/106688997/200759707-635ef6b4-f9ab-4ad1-bb6b-59a60e3a985d.png)